packages/harpguru-core/src/components/option-lock/option-lock.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  TapGestureHandler,
+  TapGestureHandlerStateChangeEvent,
+  State,
+} from 'react-native-gesture-handler'
+import { StyleSheet, View, Text } from 'react-native'
+import React from 'react'
+
+import { getSizes } from '../../styles'
+
+type OptionLockProps = {
+  readonly children: React.ReactNode
+  readonly locked: boolean
+  readonly handleTap: () => void
+}
+
+export const OptionLock = ({
+  children,
+  locked,
+  handleTap,
+}: OptionLockProps): React.ReactElement => {
+  const { 10: contentHeight } = getSizes()
+
+  const styles = StyleSheet.create({
+    fillColumn: {
+      flex: 1,
+    },
+    overlay: {
+      ...StyleSheet.absoluteFillObject,
+      flexDirection: 'row',
+      alignItems: 'center',
+    },
+    content: {
+      flex: 1,
+      alignItems: 'center',
+      justifyContent: 'center',
+      height: contentHeight,
```

Comment:
The height here is pointless because at the moment the tap handler is around the entire block, not just the limited height section. When the tap handler is applied to the limited height section it doesn't seem to work.

Reply:
https://github.com/js-jslog/harpguru/issues/3 opened to address this. I'll leave the pointless height in for now.